xds/internal/client/client_watchers_endpoints.go

```diff
@@ -91,3 +96,66 @@ func (c *Client) WatchEndpoints(clusterName string, cb func(EndpointsUpdate, err
 	})
 	return c.watch(wi)
 }
+
+// BuildEndpointsUpdate builds EndpointsUpdate.
+func BuildEndpointsUpdate(dropPercents []uint32) *EndpointsUpdate {
+	ret := &EndpointsUpdate{}
+
+	for i, d := range dropPercents {
+		drop := OverloadDropConfig{
+			Category:    fmt.Sprintf("test-drop-%d", i),
```

Comment:
These are used only in tests, right? These should be in some testutil file.

Reply:
got it